src/utils/testing.js

```diff
@@ -45,15 +44,11 @@ export function createOpenMct(timeSystemOptions = DEFAULT_TIME_OPTIONS) {
   const timeSystemKey = timeSystemOptions.timeSystemKey;
   const start = timeSystemOptions.bounds.start;
   const end = timeSystemOptions.bounds.end;
-  //we now have to set a clock
-  openmct.time.setClock('local');
 
   openmct.time.setTimeSystem(timeSystemKey, {
     start,
     end
   });
-  //we now have to set a mode
-  openmct.time.setMode(FIXED_MODE_KEY);
```

Comment:
Should we keep this?

Reply:
I didn't see any issues without it, and I had added it in so I think we're good without it